src/domains/messages-common/05_apim_v2.tf

```diff
@@ -188,8 +188,60 @@ data "azurerm_api_management_product" "apim_v2_product_services" {
   resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
 }
 
+# APIM APIs
+
+data "http" "messages_sending_external_openapi" {
+  url = "https://raw.githubusercontent.com/pagopa/io-functions-services-messages/master/openapi/index_external.yaml"
+}
+
+module "apim_v2_messages_sending_external_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v7.69.1"
+
+  name                  = format("%s-messages-sending-external-api", local.product)
+  api_management_name   = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name   = data.azurerm_api_management.apim_v2_api.resource_group_name
+  product_ids           = [data.azurerm_api_management_product.apim_v2_product_services.product_id]
+  subscription_required = true
+  service_url           = null
+
+  description  = "IO Messages Sending - External - API"
+  display_name = "IO Messages Sending - External - API"
+  path         = "api/v1/messages-sending"
+  protocols    = ["https"]
+
+  content_format = "openapi"
+  content_value  = data.http.messages_sending_external_openapi.body
+
+  xml_content = file("./api/messages-sending/v1/_base_policy.xml")
+}
+
+data "http" "messages_sending_internal_openapi" {
+  url = "https://raw.githubusercontent.com/pagopa/io-functions-services-messages/master/openapi/index.yaml"
+}
+
+module "apim_v2_messages_sending_internal_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v7.69.1"
+
+  name                  = format("%s-messages-sending-internal-api", local.product)
```

Comment:
Same thing about the new naming convention described in the [RFC](https://pagopa.atlassian.net/wiki/spaces/DevEx/pages/1032880258/RFC-DX-005+Naming+convention+nuove+risorse+Azure)